addons/web_editor/static/src/js/editor/rte.summernote.js

```diff
@@ -1211,7 +1215,7 @@ var SummernoteManager = Class.extend(mixins.EventDispatcherMixin, ServicesMixin,
      */
     _onMediaDialogDemand: function (data) {
         if (data.__alreadyDone) {
-            return;
+            throw new Error('Why the fuck do you have two of these intances?');
```

Comment:
Please... more than the comment I could have merged by inattention, I explained to you why this is needed for now... :/

Reply:
I left it in by accident, this was a debugging message for me. Sorry about that. But now that I think about it again, my system just doesn't work in the cases where this is needed, as all media dialogs will be instanciated by the first summernoteManager that registered a handler for it on the core bus, and when saving the corresponding field it will garbage collect the images of all fields on the page, and if you don't save that one then no garbage collection will happen. There is currently no way of doing this without removing the media_dialog_demand from the core.bus that I can think of.

What could be done is that the wysiwyg listens for a custom event on the root editable element and uses that to instanciate the dialogs, that way summernote can just trigger bubbling event on the image itself and the wysiwyg that's responsible for that editable can instanciate a media dialog and clean up the corresponding attachments when the field is saved or discarded. Although, while in website we have a way to discard, for HTML fields there isn't, they just don't get saved, so no garbage collection possible there...

Honestly, I feel like waiting for @ThanhDodeurOdoo 's generic garabage collection system would solve so many of these issues. (attachments are created and marked as temporary, on save we remove the temporary "flag", temporary attachments are cleaned up once a day with a cron)

Reply:
Ok, then remove garbage collection from this PR but keep the `rte.include` removal :+1: